sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/BlobTestBase.java

```diff
@@ -222,7 +222,7 @@ protected static String getAuthToken() {
             // we just need some string to satisfy SDK for playback mode. Recording framework handles this fine.
             return "recordingBearerToken";
         }
-        return new EnvironmentCredentialBuilder().build().getToken(new TokenRequestContext()
+        return new DefaultAzureCredentialBuilder().build().getToken(new TokenRequestContext()
```

Comment:
Should we use the StorageCommonTestUtils.getTokenCredentials() method here instead of using DefaultAzureCredential?